src/components/TransactionsTable/TransactionsTable.tsx

```diff
@@ -105,12 +107,32 @@ const headCells = (
 ];
 
 const TransactionsTable: React.FC<TransactionsTableProps> = ({ data }) => {
-  const [txFilter, setTxFilter] = useState('');
-  const tokenHeadCells = headCells(txFilter, setTxFilter);
+  const [txFilter, setTxFilter] = useState(-1);
+  const txHeadCells = headCells(txFilter, setTxFilter);
   const classes = useStyles();
   const { chainId } = useActiveWeb3React();
   const { palette, breakpoints } = useTheme();
   const isMobile = useMediaQuery(breakpoints.down('xs'));
+  const { t } = useTranslation();
+  const getTxString = (txn: any) => {
+    if (txn.type === TxnType.SWAP) {
+      return t('txnSwapMessage', {
```

Comment:
In the future you can put that object into a variable instead of duplicating it

Reply:
Can you tell me which object you meant to put into a variable?

Reply:
Pushed a commit with update